README.md

```diff
@@ -52,7 +48,7 @@ You should be able to get them from your favorite package manager.
   4. Build and install MLIR. Note that it uses the installation prefix specified
      in the previous step.
      ```bash
-     ninja install
+     cd llvm-project/build && ninja install llvm-project/llvm/install
```

Comment:
Why isn't `ninja install` sufficient here?

Reply:
Ninja install, only installs the libMLIRHaskell from "outer" cmake but doesn't invoke LLVM install.